cumulusci.yml

```diff
@@ -126,37 +133,25 @@ flows:
                 task: deploy_dev_config
 
     config_managed:
+        description: Configure an org for use as a dev org after package metadata is deployed.
         steps:
-            2:
-                # Configure update_admin_profile
-                options:
-                    managed: True
             3:
                 task: deploy_dev_config
                 options:
                     unmanaged: False
 
-    config_packaging:
-        steps:
-            1:
-                # Configure update_admin_profile
-                options:
-                    managed: True
-                    namespaced_org: True
-                    skip_record_types: True
```

Comment:
Can we restore this block, please? Recall that we're overriding `update_admin_profile` with a custom task that includes the `skip_record_types` option.